ingress/nginx.tmpl

```diff
@@ -0,0 +1,44 @@
+worker_processes  1;
+daemon off;
+
+error_log  /dev/stdout warn;
+pid        {{ .WorkingDir }}/nginx.pid;
+
+events {
+ worker_connections  1024;
```

Comment:
This also should be configurable - it's not clear yet what the ideal is, we'll have to capacity plan it based on expected number of persistent connections. It also depends on the number of worker processes. Although, maybe 1024 is good enough...? I'm not sure what happens if this value gets exceeded.